packages/tools/src/types/ContourSegmentationAnnotation.ts

```diff
@@ -8,7 +10,32 @@ export type ContourSegmentationAnnotationData = {
       segmentIndex: number;
       segmentationRepresentationUID: string;
     };
+    contour: {
+      /** The original polyline before livewire, to show comparison with
+       * regenerated data (eg based on spline or livewire changes).
+       */
+      originalPolyline?: Types.Point3[];
+    };
+  };
+  handles: {
+    /**
+     * Segmentation contours can be interpolated between slices to produce
+     * intermediate data.  The interpolation sources are the contour data
+     * interpolated to generate the final contour.
+     *
+     * These are sometimes required for things like livewire which need to
+     * update the handle position with a snap to nearest live point.
+     */
+    interpolationSources?: Types.PointsManager<Types.Point3>[];
```

Comment:
if this is related to the both ends that were used to interpolate contours in between, why put it under handles? this sounds like something that should be in interpolation and not handles. How about interpolationReferences instead of sources?

Reply:
It isn't for the contour, it is for the handles.  Adding some docs to update this.